### DISCUSSION TITLE
Thanks!

### DISCUSSION THREAD

**Discussion by JustAFridg** [skrysmanski/chrome-separators#14] [General]
Hey @skrysmanski, thanks for making this. 

I had this feature in my previous browser (Vivaldi) but I moved on to brave (also a browser, even more secure then Vivaldi) which doesn't have that feature so I searched for "Bookmark separator chrome" and i spotted you page. i like the idea of it and now my bookmark bar and folder looks really clean. Big thanks!

![image](https://github.com/skrysmanski/chrome-separators/assets/123732650/cc361cd2-2bd1-4cec-8a8a-108486667433)
![image](https://github.com/skrysmanski/chrome-separators/assets/123732650/d753168c-85b0-47c1-88ce-a4137591911e)
Thankful regards, JustAFridge_
(ps dont mind the stuff in the folder :joy:)

**Comment by robo042** [comment]
yes, big thanks to the author
